xla/pjrt/gpu/se_gpu_pjrt_client.cc

```diff
@@ -792,6 +792,14 @@ GetStreamExecutorGpuDeviceAllocator(
     }
   }
 
+  for (const auto& ordinal_and_device : addressable_devices) {
+    auto host_allocator =
+        GetGpuHostAllocator(ordinal_and_device.second->executor());
+    allocators.emplace_back(std::move(host_allocator),
+                            ordinal_and_device.second->compute_stream(),
+                            /*memory_space=*/5);
```

Comment:
Magic number, does it make sense to name this memory space and put it in a constant?

Reply:
@SandSnip3r Yes, I can put it in se_gpu_pjrt_client.h.  
I put "constexpr int kHostMemorySpace = 5" in runtime3/copy_thunk.h. But I don't have a good solution to just initialize it once in one header file. Do you have any suggestion?